preview_generator/preview/builder/image__wand.py

```diff
@@ -15,22 +14,11 @@
 from preview_generator.utils import imagemagick_supported_mimes
 
 
-def convert_pdf_to_jpeg(pdf: typing.IO[bytes], preview_size: ImgDims) -> BytesIO:
```

Comment:
Why do you remove this code? Wand allow to convert pdf to jpeg, why do you remove this feature?

Reply:
This code is not removed, this code has been moved to pdf builder.